demo/demo.qmlproject

```diff
@@ -16,5 +16,5 @@ Project {
         directory: "."
     }
 
-    importPaths: [ "../modules" ]
+    importPaths: [ "../modules", "../modules/Material", "../../qmlextras/modules", "../../qmlextras/modules/Material"]
```

Comment:
Why have `../modules/Material` and `../../qmlextras/modules/Material`? I thought you only needed to import the directory where modules exist, not the module directories themselves.